src/PIL/TiffImagePlugin.py

```diff
@@ -1593,6 +1594,13 @@ def _save(im, fp, filename):
     # no compression by default:
     ifd[COMPRESSION] = COMPRESSION_INFO_REV.get(compression, 1)
 
+    if im.mode == "YCbCr" and not libtiff:
+        for tag, value in {
+            YCBCRSUBSAMPLING: (1, 1),
+            REFERENCEBLACKWHITE: [15, 235, 128, 240, 128, 240],
```

Comment:
I think this should probably be by default the full range [0, 255, 128, 255, 128, 255], as that is what JPEG uses?

Reply:
I got the numbers the "CCIR Recommendation 601.1 headroom/footroom" from http://www.verypdf.com/document/tiff6/txtidx0009.htm

You think that the "no headroom/footroom" option is better though?

Reply:
Yes, 601 is for video really, while JPEG uses full range. See also https://github.com/python-pillow/Pillow/issues/4668

Reply:
So there is of course the (valid?) assumption that we're almost always dealing w/ full range image data.

Reply:
Thanks. I've updated the commit